contents/docs/product-analytics/troubleshooting.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Troubleshooting and FAQs
+---
+
+Having trouble with events? There are a few common reasons that you may not see events appear in your project.
+
+### 1. The library is being used incorrectly. (this is usually the issue)
+
+1. Visit [webhook.site](https://webhook.site/) and copy “Your unique URL”.
+2. Inside your code replace [posthog.com](https://app.posthog.com) (or [eu.posthog.com](https://eu.posthog.com) if using the EU Cloud) with “Your unique URL”.
+3. Make sure that the SDK sends a couple of events.
+4. See if the incoming events arrive as expected.
+
+![Events visible on webhook.site](../../images/docs/product-analytics/events-in-webhook.site.png)
+
+### 2. There is a bug within the library being used.
+
+#### Front-end libraries
+1. Open the devtools and switch to the network tab.
+2. Look for the request that appears to be incorrect.
+
+![network request being sent to posthog](../../images/docs/product-analytics/network-request-to-posthog.png)
+
+3. See how to [Report your specific issue](/docs/product-analytics/troubleshooting#report-your-specific-issue) to get further support.
+
+For backend libraries see the respective debugging guides, e.g. [Node.js debugging](https://posthog.com/docs/libraries/node#debugging-and-exceptions).
```

Comment:
nit: for easier readability - looked like it belonged to frontend otherwise
```suggestion
#### Backend libraries
See the respective debugging guides, e.g. [Node.js debugging](https://posthog.com/docs/libraries/node#debugging-and-exceptions).
```